src/views/virtualmachines/list/components/Timestamp/datetime.ts

```diff
@@ -0,0 +1,113 @@
+const maxClockSkewMS = -60000;
+
```

Comment:
what do u think about adding this file into utils folder?

Reply:
how about change file name to `utils.ts`?
I think we will be getting a `Timestamp` component in the future from SDK, so this file is temporary :)

Reply:
btw/ If you don't want to wait for it to be in SDK, it's also an option to put it in the components repo

Reply:
when kubevirt-components will be exposed to this project, will send this component to the repo